app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to user_post_path(user_id: @post.author_id, post_id: @post.id, id: @post.id)
+      @post.update(comments_counter: @post.comments.count)
+    else
+      redirect_to new_user_post_comment_path(user_id: @post.author_id, post_id: @post.id)
+    end
+  end
```

Comment:
Unable to view the page `http://127.0.0.1:3000/users/1/posts/4/comments/new`
![image](https://github.com/DianaBeki/Blog-app/assets/78251927/5c63bf64-88dd-4a9b-8a21-63e0d99aea7c)

- [ ] As requested by the previous reviewer, please create an action to render the templates for the comments controller.

- [ ] You need to create a `.html.erb` file within the `app/views/comments` directory. When the route takes you to that action, it also navigates you to that view. 🛠

- [ ] Similar to this, 👇 
`app/views`
![image](https://github.com/DianaBeki/Blog-app/assets/78251927/c43a1fe0-f5f0-420f-9f95-b4d1119815cd)

There should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments. 🤗